src/hooks/useActiveDescendantListFocus/__tests__/useActiveDescendantListFocus.jest.js

```diff
@@ -0,0 +1,114 @@
+import { renderHook, cleanup, act } from "@testing-library/react-hooks";
+import useActiveDescendantListFocus from "hooks/useActiveDescendantListFocus";
+import userEvent from "@testing-library/user-event";
+
+let element;
+const FIRST_ITEM_ID = "id-1";
+const SECOND_ITEM_ID = "id-2";
+const ITEM_IDS = [FIRST_ITEM_ID, SECOND_ITEM_ID];
+
+function renderHookForTest({ onItemClick = jest.fn(), isItemSelectable = () => true, isHorizontal = false }) {
+  element = document.createElement("div");
+  element.tabIndex = -1; // some tests focus the element - a tabIndex value is required for updating the document.activeIndex value
+  document.body.appendChild(element);
+
+  return renderHook(() =>
+    useActiveDescendantListFocus({
+      focusedElementRef: {
+        current: element
+      },
+      itemsIds: ITEM_IDS,
+      isItemSelectable: isItemSelectable,
+      onItemClick,
+      isHorizontalList: isHorizontal
+    })
+  );
+}
+
+function runListUnitTest(isHorizontal) {
```

Comment:
consider moving the code inside the `describe`. maybe it's just me, but I found it confusing reading `it` before reading `describe` :)